Source/Core/oneTimeWarning.js

```diff
@@ -47,5 +47,7 @@ define([
 
     oneTimeWarning.geometryOutlines = 'Entity geometry outlines are unsupported on terrain. Outlines will be disabled. To enable outlines, disable geometry terrain clamping by explicitly setting height to 0.';
 
+    oneTimeWarning.geometryZIndex = 'Entity geometry with zIndex are unsupported when the entity is dynamic, or height or extrudedHeight are defined.  zIndex will be ignored';
```

Comment:
Are dynamic entities still not supported?  This message appears to only be used in one place and doesn't check anything about dynamic.

Reply:
Oh yeah no, I fixed that.  I'll update the message